scripts/install.sh

```diff
@@ -130,21 +130,21 @@ if [[ $darwin == true ]]; then
   touch "$maestro_bash_profile"
   if ! command -v maestro > /dev/null; then
     echo "Adding maestro to your PATH in $maestro_bash_profile"
-    echo 'export PATH=$PATH:'"$MAESTRO_BIN_DIR_RAW" >> "$maestro_bash_profile"
+    echo '\nexport PATH=$PATH:'"$MAESTRO_BIN_DIR_RAW" >> "$maestro_bash_profile"
```

Comment:
This actually breaks the installation for me, as it adds \nexport PATH=$PATH:$HOME/.maestro/bin to my .zshrc.

```
➜  maestro git:(nikgraf/main) ✗ cat ~/.zshrc | grep maestro
\nexport PATH=$PATH:$HOME/.maestro/bin
```

Further, I tried adding a comment as the last line and the installation script still works for me.

Marking it as changes required so we don't land this without being sure this doesn't break any installations.